changelog.d/7631.bugfix

```diff
@@ -0,0 +1 @@
+Support the `m.login.token` login flow when OpenID Connect is enabled.
```

Comment:
s/support/advertise/ I think?

Reply:
Yeah, that's probably a better word fori t!